### ISSUE TITLE
Compatibility between knex and mock-knex

### ISSUE THREAD

**Issue by av131996** [knex/knex#4434]
#4434 #knex compatibility

Knex version: 0.95+
Database + version:
OS: 

@atiertant 

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do
 
I've updated my knex package to 0.95.4 and my tests, which are made using mock-knex failed.

2. Error message
 Unable to locate version: 0.95.4

      // Mock the connection if running jest
       if (config.mock) {
       Mock.mock(conn);
                ^
      }

Error: Unable to locate knex version 0.95.4

Is it the version 0.4.9 of mock-knex compatible with version 0.95.4 of knex?
What version of mock-knex si compatible with knex 0.95+ ?

**Comment by denisdifazio** [comment]
I don't think this is the correct place for this type of issue, but, indeed, `mock-knex` is currently not compatible with version 0.95. 
There is an open PR to update it [here](https://github.com/jbrumwell/mock-knex/pull/127)

**Comment by wolfhoundjesse** [comment]
[That PR has been merged.](https://github.com/jbrumwell/mock-knex/pull/127#issuecomment-841679837).

**Comment by mercmobily** [comment]
This has been resolved upstream in mock‑knex and does not require changes in knex.
